lib/daru/accessors/array_wrapper.rb

```diff
@@ -67,26 +67,24 @@ def dup
       end
 
       def mean
-        sum.quo(@size - @context.missing_positions.size).to_f
+        return nil if @data.compact.empty?
```

Comment:
There is `#only_valid` which does that, but it creates a new Vector and Index, which might be too expensive.